docs/docs/operator-manual/plugins/examples.mdx

```diff
@@ -0,0 +1,111 @@
+# Examples
+
+This is a bundle of example usages of the built-in Plugins. Look at [custom plugins](/operator-manual/plugins/thirdparty)
+for how to build your own resources and customizing your setup even further.
+
+## Services as `NodePort`
+
+Some load-balancers needs to address Services as LoadBalancers, e.g. the AWS Load Balancer.
```

Comment:
There is some terminology confusion here. The example has `type: NodePort` and not `type: LoadBalancer`.